### ISSUE TITLE
cant we upload folders directly to gdrive without zip?

### ISSUE THREAD

**Issue by askfriends** [bhavinamin/gdrive-upload#1]

**Comment by bhavinamin** [comment]
I don't believe the Drive API explicitly supports a sync type of upload where you specify a folder and it keeps the structure. You'd likely have to create the folder (code for which is in the supplemental common-gdrive-functions.txt file) and then recurse through the files in your local directly and upload them one-by-one into the created folder. 

It's a fine solution if you're trying to backup any static number of files or a particular file, but for user directories where anything goes, I found zipping to be easier. Hope this helps!
